pywbem/_subscription_manager.py

```diff
@@ -837,6 +841,18 @@ def add_filter(self, server_id, source_namespace, query,
             filter instance, or `None` if the `filter_id` parameter is
             specified.
 
+          source_namespace (:term:`string):
+            Optional source namespace of the indication filter. If the
+            parameter is provided the value will be inserted into the
+            CIM_IndicationFilter SourceNamespace property. Otherwise the
+            SourceNamespace property will not be created. The SourceNamespace
+            property is deprecated in the DMTF CIM_IndicationFilter class in
+            favor of SourceNamespaces( see the SourceNamespaces parameter
+            above). This parameter is provided only to support implementations
+            that use the keyword syntax with the SourceNamespace property
+            name OR to support very old WBEM servers (bprior to DMTF schema
+            version 2.22).
+
```

Comment:
I thought we said that we would not support SourceNamespace anymore??

Reply:
This seemed easy and keeps allows the old name and property at little cost.  Seemed easy way to cover the really odd  case where someone did require SourceNamespace.

Reply:
Ok, then let's keep it.